CS2Fixes.vcxproj

```diff
@@ -174,6 +174,7 @@
     <ClCompile Include="src\ctimer.cpp" />
     <ClCompile Include="src\cvars.cpp" />
     <ClCompile Include="src\detours.cpp" />
+    <ClCompile Include="src\discord.cpp" />
```

Comment:
Shouldn't the header be added as well?

Reply:
I hate how VS sometimes doesn't save this stuff...